.github/workflows/publish_reports.yml

```diff
@@ -45,5 +45,5 @@ jobs:
         uses: scacap/action-surefire-report@v1
         with:
           github_token: ${{secrets.GITHUB_TOKEN}}
-          report_paths: '${{github.workspace}}/*test-reports/**/TEST-*.xml'
+          report_paths: '${{github.workspace}}/*test-reports/*tests/**/TEST-*.xml'
```

Comment:
is it necessary?

Reply:
it was part of your PR tentative that i started to work with. i can revert now that the tests are fixed.

Reply:
cool, I was just curious, if there was a reason for it. Previous (without *tests) is more general to fit more components (all?).

Reply:
no reasons :)  you're right the previous is more general to catch up tests results.